gpytorch/models/abstract_variational_gp.py

```diff
@@ -21,14 +20,14 @@ def __init__(self, inducing_points):
         self.register_buffer("inducing_points", inducing_points)
         self.register_buffer("variational_params_initialized", torch.zeros(1))
         self.register_parameter(
-            "variational_mean",
-            nn.Parameter(torch.zeros(n_inducing)),
-            bounds=(-1e4, 1e4),
+            name="variational_mean",
+            parameter=torch.nn.Parameter(torch.zeros(n_inducing)),
+            # TODO Add prior bounds: (-1e4, 1e4)
```

Comment:
I mentioned this in my comment already, but I'd say let's just leave priors off of variational parameters for now (and remove their bounds). It's kind of strange to me to add extra terms to the variational ELBO.